StackExchange.Redis/StackExchange/Redis/RedisCommand.cs

```diff
@@ -38,6 +38,13 @@ enum RedisCommand
         FLUSHALL,
         FLUSHDB,
 
+        GEOADD,
```

Comment:
note to self: this would need to be checked in the "can I run on a slave" method; can't remember whether this is whitelist or blacklist.